python-package/lightgbm/dask.py

```diff
@@ -664,10 +664,10 @@ def __init__(
     _base_doc = LGBMClassifier.__init__.__doc__
     _before_kwargs, _kwargs, _after_kwargs = _base_doc.partition('**kwargs')
     _base_doc = (
```

Comment:
Thanks for taking these on! But I think some extra spaces have been introduced. It is also no longer necessary to use parentheses if this will be a single string.

The formatting here has to follow very very specific indentation, and if it doesn't then creating the documentation will fail with an error like this

> Warning, treated as error:
/Users/jlamb/repos/novus-lgb/docs/../python-package/lightgbm/dask.py:docstring of lightgbm.dask.DaskLGBMClassifier.__init__:67:Inline strong start-string without end-string.
make: *** [html] Error 2

Please change this section to exactly the code below (GitHub won't let me leave a suggestion that combines lines that were touched and lines that weren't, sorry)

```python
    _base_doc = f"""
        {_before_kwargs}client : dask.distributed.Client or None, optional (default=None)
        {' ':4}Dask client. If ``None``, ``distributed.default_client()`` will be used at runtime. The Dask client used by this class will not be saved if the model object is pickled.
        {_kwargs}{_after_kwargs}
        """
```

In case it is useful to you, I figure this out by rebuilding LightGBM's documentation locally following the steps at https://github.com/microsoft/LightGBM/blob/master/docs/README.rst#build.

Reply:
oh! And I should have mentioned in this comment...within f-strings, the preferred way to pad a line with leading whitespace is to use something like `{' ':4}`.

https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings